### PR TITLE
Few mup updates

### PR DESCRIPTION
**Pull request by WenkelF**
## Changelogs

- Fixed few bugs in mup implementation and added sample script for mup scaling `bash scripts/scale_mpnn.sh`
- Included option to resume traning via graphium-train `--config-path=[path to cfg] --config-name=[cfg name] +trainer.resume_from_checkpoint=[location of checkpoint] `

Some explanation on the remaining arguments in `scripts/scale_mpnn.sh`:
- Add scaling factor of model width `+architecture.mup_scale_factor=2 
- Provide base shapes from base model `+architecture.mup_base_path=mup/mpnn_base/base_shapes.yaml` (can be found at files produced by `wandb`)
- When necessary, use gradient accumulation and scale `batch_size` with the inverse to retain same effective batch size `datamodule.args.batch_size_inference=1024 datamodule.args.batch_size_training=1024 +trainer.trainer.accumulate_grad_batches=2`

---

_Checklist:_

- ~[ ] _Was this PR discussed in an issue? It is recommended to first discuss a new feature into a GitHub issue before opening a PR._~
- [ ] _Add tests to cover the fixed bug(s) or the new introduced feature(s) (if appropriate)._
- [ ] _Update the API documentation is a new function is added, or an existing one is deleted._
- [x] _Write concise and explanatory changelogs above._
- [x] _If possible, assign one of the following labels to the PR: `feature`, `fix` or `test` (or ask a maintainer to do it for you)._

---

_discussion related to that PR_

### CODE REVIEW

**Comment by WenkelF** [comment]
@DomInvivo I saw there is also an option to use a model checkpoint for the base model, which is not working correctly.

   ```
    if mup_base_path is None:
        base = model.__class__(**model.make_mup_base_kwargs(divide_factor=2))
    elif mup_base_path.endswith(".ckpt"):
        base = predictor.__class__.load_from_checkpoint(mup_base_path, map_location="cpu")
    elif mup_base_path.endswith(".yaml"):
        base = mup_base_path
    else:
        raise ValueError(f"Unrecognized file type {mup_base_path}")
```

**Review by DomInvivo — APPROVED** [review]

**Review by maciej-sypetkowski — APPROVED** [review]
